Entity-ngsiv2-datamodels-schema.json

```diff
@@ -0,0 +1,238 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-datamodels-schema.json",
+  "title": "NGSIv2 Entity. Normalized Format for FIWARE Data Models",
+  "description": "NGSIv2 Entity. Normalized Format aligned with the usage given by FIWARE Data Models. Restrictions: geo:point, geo:line, geo:polygon not covered. keyValues representation not covered",
```

Comment:
Pls add geo:box also to the list of restrictions.

Reply:
fixed